webservices/resources/committees.py

```diff
@@ -127,6 +127,96 @@ def get_committees(self, args, page_num, per_page):
         return count, committees.order_by(Committee.name).paginate(page_num, per_page, False).items
 
 
+
+
+
+
+
+
+
+
+class CommitteeByCandidate(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('state', type=str, help='Two digit U.S. State committee is registered in')
+    parser.add_argument('name', type=str, help="Committee's name (full or partial)")
+    parser.add_argument('page', type=int, default=1, help='For paginating through results, starting at page 1')
+    parser.add_argument('per_page', type=int, default=20, help='The number of results returned per page. Defaults to 20.')
+    parser.add_argument('committee_type', type=str, help='The one-letter type code of the organization')
+    parser.add_argument('designation', type=str, help='The one-letter designation code of the organization')
+    parser.add_argument('organization_type', type=str, help='The one-letter code for the kind for organization')
+    parser.add_argument('party', type=str, help='Three letter code for party')
+    parser.add_argument('year', type=str, default=None, help='A year that the committee was active- (fter original registration but before expiration.)')
+
+    @marshal_with(committee_list_fields)
+    def get(self, **kwargs):
+
+        args = self.parser.parse_args(strict=True)
+
+        # pagination
+        page_num = args.get('page', 1)
+        per_page = args.get('per_page', 20)
+
+        count, committees = self.get_committees(args, page_num, per_page, **kwargs)
+
+        data = {
+            'api_version': '0.2',
+            'pagination': {
+                'page': page_num,
+                'per_page': per_page,
+                'count': count,
+                'pages': int(count / per_page),
+            },
+            'results': committees
+        }
+
+
+        return data
+
+    def get_committees(self, args, page_num, per_page, **kwargs):
```

Comment:
Seems like the `get_committees` method from `CommitteeList` could be pretty readily modified to take a candidate_id and give back the results we want, eliminating the need for this method (and perhaps even this whole Resource, but I'm not sure on that yet...)